plugin/configuration.py

```diff
@@ -73,6 +74,33 @@ def _on_done(self, index):
             self.window.status_message("{} enabled, starting server...".format(config_name))
 
 
+class LspExecuteCommand(LspTextCommand):
```

Comment:
Maybe this class should live is own file?
I would suggest creating a `plugin/execute_command.py` file.